spec/doctor_teeth_spec.rb

```diff
@@ -3,27 +3,33 @@
 require 'spec_helper'
 
 describe Sinatra do
-  it "should allow accessing the home page" do
+  it 'should allow accessing the home page' do
```

Comment:
interesting.  we can turn it off in this repo if we think it's too pedantic...

Reply:
I just wanted to share my opinion. :)  Since at the moment you are writing the code.  You get to make the call. :)  When I start a repo I will turn it off for my code. :)